types/two.js/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://two.js.org
 // Definitions by: Carlos Precioso <https://github.com/cprecioso>
 //                 Konstantin <https://github.com/demkonst>
+// 				   Levente <https://github.com/leventefabry>
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/two.js/index.d.ts;
  Line: 5;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.